js/dydx.js

```diff
@@ -0,0 +1,1252 @@
+                    },
+                    'post': {
+                        'testnet/tokens': 172800,
+                        'onboarding': 60,
+                        'transfers': 60,
+                        'api-keys': 60,
+                        'accounts': 60,
+                        'withdrawals': 60,
+                        'fast-withdrawals': 60,
+                        'orders': 1, // variable
+                    },
+                    'delete': {
+                        'api-keys': 60,
+                        'orders': 1, // variable, 3 requests per 10 seconds PER SYMBOL
+                        'orders/{id}': 1, // variable, 250 requests per 10 seconds PER SYMBOL
+                        'active-orders': 1, // variable
+                    },
+                    'put': {
+                        'users': 60,
+                        'emails/send-verification-email': 3000,
+                    },
+                },
+            },
+            'commonCurrencies': {
+                //
+            },
+            'precisionMode': TICK_SIZE,
+            'fees': {
+                'trading': {
+                    'tierBased': false, // complicated tier system per coin
+                    'percentage': true,
+                    'maker': 0.05 / 100,
+                    'taker': 0.1 / 100,
+                },
+            },
+            'exceptions': {
+                'exact': {
+                    '404': ExchangeNotAvailable,
+                    '403': AuthenticationError,
+                    '400': ExchangeError,
+                    'dydx-api-key': AuthenticationError,
+                    'dydx-passphrase': AuthenticationError,
+                    'dydx-timestamp': BadRequest,
+                },
+                'broad': {
+                    'must be a valid string that is less than length:': BadRequest,
+                    'Cannot read property ': BadRequest,
+                    'signature must be a string': AuthenticationError,
+                    'must be a boolean': BadRequest,
+                    'Invalid number': BadRequest,
+                    'expiration must be an ISO8601 string': BadRequest,
+                    'Invalid signature for order': InvalidOrder,
+                    'No order for market: ': InvalidOrder,
+                    'status must be a valid array of subset': BadRequest,
+                    'API key not found': AuthenticationError,
+                    'api-key must be a valid UUID in headers': AuthenticationError,
+                    'Unauthorized': AuthenticationError,
+                    'passphrase must be a valid base64url string in headers that is 20 characters long': AuthenticationError,
+                    'timestamp must be a valid ISO string in headers': BadRequest,
+                    'ethereum address must be a valid string in headers': AuthenticationError,
+                    // old
+                    'See /corsdemo for more info': AuthenticationError,
```

Comment:
This error is most likely generated by a CORS-server, not by the dYdX API. I would suggest removing line 185.